website/forms/php/filters/general/scripts-dependency.md

```diff
@@ -2,3 +2,12 @@
 id: scripts-dependency
 title: Scripts dependency
 ---
+
+In case you are having a conflict with some other plugin or theme, you can try to change the order of the scripts loading. This filter will make forms scrips dependent on your scrips and load them after your scripts.
```

Comment:
```suggestion
In case you are having a conflict with some other plugin or theme, you can try to change the order of the scripts loading. This filter will make forms scripts dependent on your scripts and load them after your scripts.
```